resume-buddy/src/main/java/com/google/sps/data/User.java

```diff
@@ -29,5 +30,27 @@ public User(
     this.career = career;
     this.schoolYear = SchoolYear.valueOf(year.toUpperCase());
     this.matchID = matchID;
+    this.isCurrentUser = false;
```

Comment:
Instead of filling out all the member variables, you could just use the constructor below
```
return this(fname, lname, email, school, career, degree, year, matchID, false);
```